src/main/java/seedu/academyassist/logic/commands/EditCommand.java

```diff
@@ -130,14 +135,14 @@ public boolean equals(Object other) {
         }
 
         EditCommand otherEditCommand = (EditCommand) other;
-        return nric.equals(otherEditCommand.nric)
+        return studentId.equals(otherEditCommand.studentId)
                 && editPersonDescriptor.equals(otherEditCommand.editPersonDescriptor);
     }
 
     @Override
     public String toString() {
         return new ToStringBuilder(this)
-                .add("NRIC", nric)
+                .add("NRIC", studentId)
```

Comment:
Same here, the parameter here is a bit confusing.

Reply:
My bad :(